nunit.tests.core.csharp/nunit.tests.core.csharp.csproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
+    <IsPackable>false</IsPackable>
```

Comment:
I always delete this 🤷‍♂️

Reply:
Line 5 you mean ?

Reply:
Does it make sense to pack a test project?  Or should we just 'don't care' on it ?

Reply:
Right. It doesn't make sense to pack a test project, but I only pack specific projects so this property would never have an effect. It might help folks that `dotnet pack` an entire solution rather than specific projects. So I'm not saying we necessarily should delete it, but anecdotally the need for it seems rare.

Reply:
Ok, I'll skip it